src-built-in/components/assets/css/_variables.css

```diff
@@ -1,142 +1,203 @@
+    --titlebar-tab-windowFocused-active-border-color: var(--slate2); 
+    --titlebar-tab-active-background-color: var(--slate4); /* Specifies the background color of the active tab in an inactive window. */
+    --titlebar-tab-active-border-color: var(--slate3);
+    --titlebar-tab-inactive-background-color: var(--slate7); /* Inactive tabs are the same color for active and inactive windows. */
+    --titlebar-tab-inactive-border-color: var(--slate3);
+
+    --titleBar-button-hover-active-color: var(--blue6);
+    --titleBar-button-hover-inactive-color: var(--blue7);
+    --titleBar-button-highlight-active-color: var(--blue6);
+    --titleBar-button-highlight-inactive-color: var(--blue7);
+    --titleBar-button-hover-linker-group: var(--gray3);
+
+    --titleBar-text-color: var(--font-color);
+    --titlebar-text-active-color: var(--font-color);
+    --titleBar-text-inactive-color: var(--gray3);
+    --titleBar-font-weight: 300;
+    --titleBar-font-size: 12px;
+
+
+
+    /* Dialogs */
+    --dialog-background-color: var(--slate7);
+    --dialog-font-color: #FFFFFF;
+    --dialog-font-weight: 300;
+    --dialog-body-padding: 32px;
+    --dialog-input-background-color: var(--slate7);
+    --dialog-input-border-color: var(--gray6);
+    --dialog-input-border-focus-color: var(--gray);
+
+    /* Buttons */
+    --button-affirmative-fill: var(--blue6);
+    --button-affirmative-border: var(--blue5);
+    --button-affirmative-border-hover: var(--blue2);
+    --button-neutral-fill: var(--gray2);
+    --button-neutral-border: var(--gray6);
+    --button-neutral-border-hover: var(--gray2);
+    --button-negative-fill: var(--red5);
+    --button-negative-border: var(--red3);
+    --button-negative-border-hover: var(--red1);
+
+    --button-neutral-solid-text: #000000;
+    --button-solid-text: #FFFFFF;
+
+    /* Menus */
+    --menu-font-size: 14px;
+    --menu-background-color: var(--slate6);
+    --menu-primary-font-color: #FFFFFF;
+    --menu-primary-background-color: var(--slate6);
+    --menu-primary-hover-color: var(--slate3);
+    --menu-secondary-background-color: var(--slate4);
+    --menu-secondary-hover-color: var(--slate3);
+
+    --menu-pin-background-color: var(--slate1);
+    --menu-pin-background-hover-color: var(--slate2);
+    --menu-pin-border-color: var(--slate1);
+    --menu-delete-background-color: var(--red1);
+
+    --scrollbar-color: var(--slate1);
+
+    /* Search Menu */
+    --search-result-active-item-color: var(--slate3);
+    --search-section-max-height: 30px;
     --search-input-height:var(--toolbar-height);
     --search-input-line-height: var(--toolbar-height);
+
+    /* Window Frame*/
+    --window-frame-inactive-color: var(--slate);
+    /* --window-frame-active-color: var(--window-frame-inactive-color); */
+
+    /* Window styles*/
+    --scrim-overlay: var(--blue7);
+    --scrim-negative-overlay: var(--red7);
+    
+    /* Complex menus (e.g., Preferences and App Catalog) */
```

Comment:
@DanNicolai we have about one minute in finsemble-time to come up with a better naming convention than "complex-nav" before it's cemented in stone and we can never change it. Perhaps simply, "nav"?